TheWarWithin/Priorities/MageFrost.simc

```diff
@@ -43,7 +43,7 @@ actions.aoe_ss+=/blizzard,if=talent.ice_caller|talent.freezing_rain|active_enemi
 actions.aoe_ss+=/shifting_power,if=cooldown.icy_veins.remains>10&(fight_remains+10>cooldown.icy_veins.remains)
 actions.aoe_ss+=/glacial_spike,if=buff.icicles.stack=5&(cooldown.flurry.up|remaining_winters_chill|active_enemies<5&freezable&cooldown.ice_nova.ready&!buff.fingers_of_frost.up)
 actions.aoe_ss+=/ice_lance,if=buff.fingers_of_frost.up&!prev_gcd.1.glacial_spike|remaining_winters_chill
-actions.aoe_ss+=/flurry,if=remaining_winters_chill=0
+actions.aoe_ss+=/flurry,if=remaining_winters_chill=0&(!buff.fingers_of_frost.react|buff.fingers_of_frost.remains>1.5)&(!talent.freezing_winds|!buff.icy_veins.up|buff.fingers_of_frost.remains>2)
```

Comment:
We try to convert `react` to the statements we normally use.

In this case for a buff, you can simply use `buff.fingers_of_frost.down` or `!buff.fingers_of_frost.up`. `react` is specific to simc such that it incorporates the "human factor" of reaction time. That already exists in the addon because it is piloted by humans (allegedly)